resume.html

```diff
@@ -5,6 +5,9 @@
 		Resume: MICHAEL DENMAN
 	</title>
 	<style>
+		* {
+			box-sizing: border-box;
+		}
 		body {
 			background-color: #333333;
 			color: #cccccc;
```

Comment:
Let's change these up to only style elements on your page.  Reserve body-wide styles for last.